content/en/docs/concepts/security/hardening-guide/scheduler.md

```diff
@@ -0,0 +1,91 @@
+| `tls-cipher-suites`            | Always provide a list of preferred cipher suites. This ensures encryption never happens with insecure cipher suites.                                                                                                               |
+{{</table>}}
+
+## Scheduling configurations for custom schedulers
+This section covers security hardening when using custom schedulers based on the Kubernetes
+scheduling code.
+The cluster administrator needs be careful with the plugins that use the `queueSort`, `filter`, or permit [extension points](https://github.com/docs/reference/scheduling/config/#extension-points).
+Scheduling happens in a series of stages that are exposed through the extension points.
+As a cluster administrator, you can enable plugins that define their own extension points.
+Doing so can affect the defined scheduling behaviors of the kube-scheduler in your cluster.
+
+When using a custom scheduler, plugin extension points such as `queueSort`, `filter` and `permit` should be used with care.
+
+Exactly one plugin that uses the `queueSort` extension point can be enabled at a time. Any plugins that use `queueSort` should be scrutinized.
+
+Plugins that implement the `filter` extension point can potentially mark all nodes as unschedulable. This can bring scheduling of new pods to a halt.
+
+Plugins that implement the `permit` extension point  can prevent or delay the binding of a Pod. Such plugins should be thoroughly reviewed by the cluster administrator.
+
+When using a plugin that is not one of the [default plugins](https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins), consider disabling the `queueSort`, `filter` and `permit` extension points as follows:
+
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1
+kind: KubeSchedulerConfiguration
+profiles:
+  - schedulerName: default-scheduler
+  - schedulerName: my-custom-scheduler
+    plugins:
+      queueSort:
+        disabled:
+        - name: '*'
+      filter:
+        disabled:
+        - name: '*'
+      permit:
+        disabled:
+        - name: '*'
+```
+This creates two scheduler profiles: `default-scheduler` and ` my-custom-scheduler`.
+Whenever the `.spec` of a Pod does not have a value for `.spec.schedulerName`,
+the kube-scheduler runs for that Pod, using its main configuration, and default plugin.
+If you define a Pod with `.spec.schedulerName` set to `my-custom-scheduler`, the
+kube-scheduler also runs but with a custom configuration; in that custom configuration,
+the  `queueSort`, `filter` and `permit` extension points are disabled.
+If you use this kube-scheduler configuration, and don't run any custom scheduler, and
+you then define a Pod with  `.spec.schedulerName` set to `nonexistent-scheduler` (or
+any other scheduler name that doesn't exist in your cluster), then the Pod stays pending scheduling. You would see a `FailedScheduling` event for the Pod and could manually
```

Comment:
> You would see a `FailedScheduling` event for the Pod

This is not accurate. A pod specifies non-existent schedulerName would not see any event.